wandb/__init__.pyi

```diff
@@ -416,48 +416,96 @@ def log(
     of the W&B app, or export data to visualize and explore locally, e.g. in
     Jupyter notebooks, with [our API](https://docs.wandb.ai/guides/track/public-api-guide).
 
-    In the UI, summary values show up in the run table to compare single values across runs.
-    Summary values can also be set directly with `wandb.run.summary["key"] = value`.
-
     Logged values don't have to be scalars. Logging any wandb object is supported.
-    For example `wandb.log({"example": wandb.Image("myimage.jpg")})` will log an
+    For example `run.log({"example": wandb.Image("myimage.jpg")})` will log an
     example image which will be displayed nicely in the W&B UI.
     See the [reference documentation](https://docs.wandb.com/ref/python/data-types)
     for all of the different supported types or check out our
     [guides to logging](https://docs.wandb.ai/guides/track/log) for examples,
     from 3D molecular structures and segmentation masks to PR curves and histograms.
-    `wandb.Table`s can be used to logged structured data. See our
+    You can use `wandb.Table` to log structured data. See our
     [guide to logging tables](https://docs.wandb.ai/guides/data-vis/log-tables)
     for details.
 
     Logging nested metrics is encouraged and is supported in the W&B UI.
-    If you log with a nested dictionary like `wandb.log({"train":
-    {"acc": 0.9}, "val": {"acc": 0.8}})`, the metrics will be organized into
+    When you log with a nested dictionary like `run.log({"train":
+    {"acc": 0.9}, "val": {"acc": 0.8}})`, the metrics are organized into
     `train` and `val` sections in the W&B UI.
```

Comment:
is this even correct? i think backend just flatten everything nested if i remember correctly :(

Reply:
Good catch, nesting only happens with slash-separated metric names, like

```
run.log({
    "train/acc": 0.9,
    "val/acc": 0.8,
})
```

It's surprising that we don't do that with nested metrics... I'll document this, but I don't like it.

Reply:
Updated

Reply:
agreed... the nested metrics actually make more sense to be nested in the UI... at least intuitively that what i would expect